scripts/deploy-infrastructure-dev.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
This script will run the `deploy-infrastructure-dev` job on a local machine through the [CircleCI local CLI](https://circleci.com/docs/2.0/local-cli/)